src/modules/formBuilder/components/itemEditor/conditionals/EnableWhenCondition.tsx

```diff
@@ -254,6 +259,21 @@ const EnableWhenCondition: React.FC<EnableWhenConditionProps> = ({
                   required
                 />
               ))}
+            {answerInputType === 'answerDate' && (
+              <Controller
```

Comment:
Noting that this doesn't have shouldUnregister. Just flagging as a potential for merge conflicts with https://github.com/greenriver/hmis-frontend/pull/899